VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/TransactionsList.razor

```diff
@@ -1,294 +1,122 @@
-                                        else if (txd.FromSubAccount)
-                                        {
-                                            <td>Sub</td>
-                                        }
-                                        else 
-                                        { 
-                                            <td>Out</td>
-                                        }
-                                        <td class="text-wrap" style="max-width:80px;">@txd.Receiver</td>
-                                        <td class="text-wrap" style="max-width:60px;">@txd.Time</td>
-                                        <td class="text-center" @onclick="async () => ShowTxDetails(txd.Info.Txid)"><i class="oi oi-info" style="font-size:18px;"></i></td>
-                                    </tr>
-                                }
-                            }
-                            @if (maxLoadedTxs < Transactions.Count)
-                            {
-                                var diff = Transactions.Count - maxLoadedTxs;
-                                @if (diff < 10)
-                                {
-                                    <tr><td colspan="4"> <button class="btn btn-secondary" @onclick="loadNextTx">Show Next @diff transactions</button></td></tr>
-                                }
-                                else
-                                {
-                                    <tr><td colspan="4"> <button class="btn btn-secondary" @onclick="loadNextTx">Show Next 10 transactions</button></td></tr>
-                                }
-                            }
-                        </tbody>
-                    </table>
+                    <VENFTApp_Blazor.Components.Table ItemType="TxDetails" Items="TransactionsDetails">
+                        <VENFTApp_Blazor.Components.Column ItemType="TxDetails" Title="Way">
+                            @context.Way
+                        </VENFTApp_Blazor.Components.Column>
+                        <VENFTApp_Blazor.Components.Column ItemType="TxDetails" Title="Receiver">
+                            @context.Receiver
+                        </VENFTApp_Blazor.Components.Column>
+                        <VENFTApp_Blazor.Components.Column ItemType="TxDetails" Title="Date">
+                            @context.Time
+                        </VENFTApp_Blazor.Components.Column>
+                        <VENFTApp_Blazor.Components.Column ItemType="TxDetails" Title="Details">
+                            <button class="btn btn-primary" @onclick="() => ShowTxDetails(context.Info.Txid)">
+                                Details
+                            </button>
+                        </VENFTApp_Blazor.Components.Column>
+                    </VENFTApp_Blazor.Components.Table>
+                    <VENFTApp_Blazor.Components.Pagination Total="@Transactions.Count" PerPage="@PerPage" OnChange="@HandlePaginationChanged" />

```

Comment:
We use global using for AntDesign and there are naming collisions. This should looks like:
```
<Table ItemType="TxDetails" Items="TransactionsDetails">
    <Column ItemType="TxDetails" Title="Way">
        @context.Way
    </Column>
...
</Table>
<Pagination Total="@Transactions.Count" PerPage="@PerPage" OnChange="@HandlePaginationChanged" />
```

It is also possible to define `ItemType` only in `Table`, but only in newest ASP.NET Core 6

Reply:
Yes, I agree. Theoretically we can make wrappers for AntDesign components that we are using right now and replace them later.

Reply:
@fyziktom shall we start on it?

Reply:
good point. yes please. I think it is logical step before refactoring of other parts of UI.